packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -17,21 +17,13 @@ test.describe('Firefox Desktop Sync v3 sign in', () => {
   test('verified, does not need to confirm', async ({ target }) => {
     const { page, login, connectAnotherDevice, signinTokenCode } =
       await newPagesForSync(target);
-    const uaStrings = {
```

Comment:
I honestly don't know what this tests but it is over 50 versions old. I think it is safe to remove...